electrum/lnworker.py

```diff
@@ -1325,3 +1326,55 @@ def current_feerate_per_kw(self):
         if feerate_per_kvbyte is None:
             feerate_per_kvbyte = FEERATE_FALLBACK_STATIC_FEE
         return max(253, feerate_per_kvbyte // 4)
+
+    @log_exceptions
+    async def htlc_switch(self):
```

Comment:
note that
- an await re a single peer can ~arbitrarily delay processing for another peer
- an exception raised by any peer that propagates out to here, will kill the task (and hence lnwallet)

Reply:
thanks for the review. your first point should be addressed by the latest commit.

Reply:
Regarding your second point, I realize that the htlc_switch task could be distributed to peers, and moved to lnpeer. When I started working on this, I was under the impression that this task would need to be centralized, but that does not seem to be the case anymore.